### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by PromzzyKoncepts**
# Hey There 👋🏻 
> In this PR, you can find that I:

- [x] Styled the different components
- [x] Styled the redux components
- [x] added transitions and effects

## Happy Reviewing

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -31,7 +26,7 @@ const Book = (obj) => {
         <div className="functions">
           <p>Comments</p>
           <hr />
-          <button style={button} type="button" onClick={() => RemoveBtn(id)}>Remove</button>
+          <button className="button" type="button" onClick={() => RemoveBtn(id)}>Remove</button>
```
Comment (arslanbisharat) [inline src/components/Book.js:29]: - [ ] The the design there is a circle present to represent the completion percentage. Please make it in your design too. 

Your Design: 

image

Original 

image

File: src/style.css [hunk 2]
```diff
@@ -71,12 +71,33 @@ h1 {
   font-size: 0.85rem;
 }
 
+.author:focus {
+  outline: 1px solid;
+  border-radius: 4px;
+}
+
```
Comment (arslanbisharat) [inline src/style.css:84]: - [ ] The font of the text should match with the design. Please make sure you follow the design.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
Hi @PromzzyKoncepts, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- The Pull Request has a proper title and description. ✔️
- You have added a descriptive Readme file. Good Job. ✔️
- There is no linter error is present. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Please make sure you right descriptive comments. Commenting involves placing Human Readable Descriptions inside of computer programs detailing what the Code is doing. Proper use of commenting can make code maintenance much easier, as well as helping make finding bugs faster. Further, commenting is very important when writing functions that other people will use. :x:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lordkaito — APPROVED** [review]
Hi @PromzzyKoncepts, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**